docs/tutorials/mint-locally.md

```diff
@@ -64,7 +64,7 @@ await registry.addMember(1, empCreator.address)
 7. We also need to register the collateral token with the `collateralTokenWhitelist`.
 
 ```js
-collateralTokenWhitelist = await AddressWhitelist.at(await empCreator.collateralTokenWhitelist())
+collateralTokenWhitelist = await AddressWhitelist.deployed()
```

Comment:
Unrelated to this PR, but shouldn't this be const?
```suggestion
const collateralTokenWhitelist = await AddressWhitelist.deployed()
```

Reply:
yes great catch